src/swagger.js

```diff
@@ -0,0 +1,23 @@
+import path from 'path';
+
+const routesPath = path.resolve(process.cwd(), __dirname, '../routes/index.js');
+
+const config = {
+  info: {
+    title: '1kbIdeas',
+    version: '1.0.0', // Version (required)
+    description: 'The one-stop shop for all the self-help ideas you need',
+    termsOfService: '',
+    contact: {
+      email: ''
+    },
+    license: {
+      name: 'Apache 2.0',
+      url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+    }
+  },
+  basePath: '/api/v1',
+  apis: [routesPath]
+};
+
+module.exports = config;
```

Comment:
Remove this and use `export const config = ...` on line 5

Reply:
<img width="1440" alt="Screenshot 2019-07-25 at 10 09 58 AM" src="https://user-images.githubusercontent.com/31247452/61861772-8635b300-aec4-11e9-897a-612baa164807.png">
Swagger keep giving 'Definition file should contain an info object!' when I use es6 export